### ISSUE TITLE
Plugins can't see most of the filesystem

### ISSUE THREAD

**Issue by bandithedoge** [robbert-vdh/yabridge#291]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

All plugins I've tried have had some issue related to missing files, even though they are definitely there. This ranges anywhere from presets to UI resources or additional DLLs, so it obviously breaks most plugins. When browsing through files (like from a plugin's preset browser) there are entire directories missing. It happens only when running plugins through yabridge, executables run with Wine have no issues (including Windows DAWs trying to run the same plugins).

### What did you expect to happen?

Plugins can access all the files.

### What actually happened?

They don't. Duh.

### Operating system

NixOS 24.05

### How did you install yabridge?

nixpkgs

### yabridge version

5.0.5

### yabridgectl version

5.0.5

### Wine version

wine-ge 8.25

### Plugin

*

### Plugin type

both VST2 and VST3

### Plugin architecture

both 64-bit and 32-bit

### Host

Bitwig Studio 5.0.4, Ardour 8.1.0, REAPER 7.05

### Debug log

[yabridge.log](https://github.com/robbert-vdh/yabridge/files/13745183/yabridge.log)

**Comment by bandithedoge** [comment]
I've also tried mapping a directory as a virtual drive in Wine and the drive doesn't show up either.

**Comment by robbert-vdh** [comment]
Are you running those standalones with the same Wine 8.0 version used for yabridge? There's probably some difference in the environment used for yabridge and for the standalones.

**Comment by bandithedoge** [comment]
Yes, I'm using the same Wine binaries. I even tried setting `WINELOADER`.

**Comment by robbert-vdh** [comment]
You're saying it happens with all plugins. Which plugins are those? This issue seems to be isolated to your setup, so there has to be something specific about your setup or the plugins you're using.

**Comment by bandithedoge** [comment]
Here's a few I just checked:

- Superior Drummer 3 - GUI only shows a message saying "no resources seem to have been installed"; EZbass seems to work fine, though
- Serum - GUI just doesn't open
- Kontakt 7 - crashes yabridge-host.exe and hangs the entire audio system

The following work but still can't see most of the filesystem (ie. from a preset browser):

- all Acon Digital Restoration Suite plugins
- Eventide Blackhole
- Excite Audio Lifeline Console
- FabFilter Pro-Q 3
- Neural DSP Archetype Nolly
- SSL Native X-Saturator
- Valhalla Vintage Verb

I double-checked all of these in Ableton Live with the same Wine binaries and they work just fine.

**Comment by robbert-vdh** [comment]
Could you try installing some of them in a clean prefix, without using Bottles? It sounds like those plugins are installed outside of a Wine prefix and the default `~/.wine` prefix doesn't have the required files, but that's not the case for the log you posted.

**Comment by bandithedoge** [comment]
I made a new prefix outside of Bottles and installed Kontakt, SD3 and Serum since I still had the installers for those. While Serum crashed my window manager after asking for a serial key, Kontakt and SD3 seem to work just fine.

I tried to import this new prefix into Bottles by copying `bottles.yml` from the old one and reinstalling SD3 through Bottles - no issues still. This pretty much confirms that the problem is somehow connected to the way Bottles sets up new prefixes.

**Comment by JoelNichols** [comment]
This is because of the sandboxing applied to flatpaks (i.e. bottles).

Use Flatseal and give the Bottles flatpak access to the drives / folders you need and it *should* then work.

**Comment by bandithedoge** [comment]
> This is because of the sandboxing applied to flatpaks (i.e. bottles).
> 
> Use Flatseal and give the Bottles flatpak access to the drives / folders you need and it _should_ then work.

I'm not using Flatpak.
